app/src/main/java/org/gdg/frisbee/android/activity/AppInviteDeepLinkActivity.java

```diff
@@ -43,10 +43,11 @@
     private Intent mCachedInvitationIntent;
 
     @Override
-    protected void createGoogleApiClient() {
-        mGoogleApiClient = new GoogleApiClient.Builder(this)
-                .addApi(AppInvite.API)
-                .build();
+    protected GoogleApiClient createGoogleApiClient() {
+        GoogleApiClient googleApiClient = new GoogleApiClient.Builder(this)
```

Comment:
The app invite can happen before the user gets a chance to sign into the
app as it is a deep link
On 26 Jan 2016 11:44, "Said Tahsin Dane" <notifications@github.com> wrote:

> In
> app/src/main/java/org/gdg/frisbee/android/activity/AppInviteDeepLinkActivity.java
> <https://github.com/gdg-x/frisbee/pull/547#discussion_r50819120>:
>
> > @@ -43,10 +43,11 @@
> >      private Intent mCachedInvitationIntent;
> >
> >      @Override
> > -    protected void createGoogleApiClient() {
> > -        mGoogleApiClient = new GoogleApiClient.Builder(this)
> > -                .addApi(AppInvite.API)
> > -                .build();
> > +    protected GoogleApiClient createGoogleApiClient() {
> > +        GoogleApiClient googleApiClient = new GoogleApiClient.Builder(this)
>
> This can be inlined. And one question: we don't need the other Apis here
> right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/547/files#r50819120>.
>